<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1355</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>887</x>
      <y>-30</y>
      <width>20</width>
      <height>801</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pwrBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ON / OFF</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="NOKindicator">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt; color:#5e5c64;&quot;&gt;OFF&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="light1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>298</x>
      <y>368</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/light2</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Loop">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>761</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/bkg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="light2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>202</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/light2</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="light3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>438</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/light2</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="light4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>599</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/light2</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="light6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>618</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/light2</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="light5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>471</x>
      <y>475</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/light2</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Normal - Adult</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ventricular Fibrillation - Adult</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ventricular Tachycardia - Adult</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Normal - Child</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ventricular Fibrillation - Child</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ventricular Tachycardia - Child</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Flatline</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>250</y>
      <width>285</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="toggleCPR">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>40</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Toggle Giving CPR</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>10</y>
       <width>150</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
     <property name="text">
      <string>CPR TESTING</string>
     </property>
    </widget>
    <widget class="QPushButton" name="shallowCPRBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>117</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Toggle Shallow</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>84</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Compression Depth</string>
     </property>
    </widget>
    <widget class="QPushButton" name="adequateCPRBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>120</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Toggle Adequate</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="LCD">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>260</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #b0b0b0;
border-radius: 5px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="shocksLabel">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>6</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SHOCKS:</string>
     </property>
    </widget>
    <widget class="QLabel" name="shockCounter">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>6</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="timerLabel">
     <property name="geometry">
      <rect>
       <x>171</x>
       <y>6</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="CPRDepthBar">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>30</y>
       <width>25</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar{
	border: 2px solid rgb(61,56,70);
	border-radius: 0px;
}

QProgressBar::chunk {
	background-color: rgb(61, 56, 70)
}</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="minCPRLine">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>76</y>
       <width>25</width>
       <height>3</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(61, 56, 70);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="maxCPRLine">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>90</y>
       <width>25</width>
       <height>3</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(61, 56, 70);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="LCD_text">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="batteryBar">
     <property name="geometry">
      <rect>
       <x>104</x>
       <y>5</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>95</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>80</y>
      <width>285</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>10</y>
       <width>220</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
     <property name="text">
      <string>DEFIB PAD TESTING</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>84</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Pad Placement</string>
     </property>
    </widget>
    <widget class="QPushButton" name="padsGood">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>115</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Place Correctly</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="padsBad">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>115</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Placed Incorrectly</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="adultPadBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Adult Pads</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pedPadBtn">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>40</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Pediatric Pads</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <zorder>line</zorder>
   <zorder>pwrBtn</zorder>
   <zorder>NOKindicator</zorder>
   <zorder>Loop</zorder>
   <zorder>light1</zorder>
   <zorder>light2</zorder>
   <zorder>light3</zorder>
   <zorder>light4</zorder>
   <zorder>light6</zorder>
   <zorder>light5</zorder>
   <zorder>comboBox</zorder>
   <zorder>groupBox</zorder>
   <zorder>LCD</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1355</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAED">
    <property name="title">
     <string>AED</string>
    </property>
   </widget>
   <addaction name="menuAED"/>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
